### ISSUE TITLE
Elm Package Install Failing with Prelude.Read Error

### ISSUE THREAD

**Issue by NeilC** [elm-lang/elm-platform#134]
Elm package has suddenly begun throwing the following error when running the install command in a directory with or without an elm-package.json file.

`Error: failed with 'Prelude.read: no parse' when sending request to   <http://package.elm-lang.org/all-packages?elm-package-version=0.16&since=elm-package.exe: Prelude.read: no parse`

I am able to visit the url listed in the error with browser and also download the data from it in the console using powershell. I've tried re-installing and adding firewall rules to no avail.

**Comment by jvoigtlaender** [comment]
From the looks of it, I'm pretty sure the problem is not downloading from the URL, but actually creating the URL. It looks like `elm-package` is trying to add a value for the `since=` argument. While doing that, it encounters a parse error. That is, it is unable to parse the system time into something that makes sense for it. In fact, I'll wager that the error you are encountering comes from this line: https://github.com/elm-lang/elm-package/blob/7a614594183471530bf415b03b0f57d6f959f322/src/Store.hs#L62. There, a string that was read from a file named `last-updated` is parsed as a time stamp. So, the questions are:
* Does that file (still) exist on your system?
* Does it (suddenly, for some reason) not contain something that makes sense to Haskell's `read` function as a time stamp?

**Comment by NeilC** [comment]
Direct hit @jvoigtlaender =) I deleted the elm folder in roaming and everything is now working normally. 

I had found the `last-updated ` file in `C:\Users\user\AppData\Roaming\elm` and after trying to open it in VSCode was told that it was in **binary format**. I tried to delete it but was told the file was in use. Using LockHunter, a program for unlocking files on Windows, I saw 5 instances of VS Code locking the file, while in my taskbar there were none. this is also after a reboot. I unlocked it, deleted it and all is good now. The last-updated file now contains a UTC date time string. 

I have a feeling this has something to do with the elm tools for vs code found [here](https://github.com/sbrink/vscode-elm) but I can't confirm it, it's just a hunch.
